test/testpaint/Compat.cpp

```diff
@@ -36,52 +36,37 @@ uint32_t gCurrentViewportFlags;
 uint32_t gScenarioTicks;
 uint8_t gCurrentRotation;
 
-const CoordsXY CoordsDirectionDelta[] = {
-        { -32,   0 },
-        {   0, +32 },
-        { +32,   0 },
-        {   0, -32 },
-        { -32, +32 },
-        { +32, +32 },
-        { +32, -32 },
-        { -32, -32 }
-};
+const CoordsXY CoordsDirectionDelta[]
+    = { { -32, 0 }, { 0, +32 }, { +32, 0 }, { 0, -32 }, { -32, +32 }, { +32, +32 }, { +32, -32 }, { -32, -32 } };
```

Comment:
I'd much prefer vertical lists. These are far easier to read IMO.

Reply:
A trailing comma should do the trick.